toggle.css

```diff
@@ -0,0 +1,30 @@
+.switch {
+  position: relative;
+  display: inline-block;
+  width: 60px;
+  height: 30px;
+  background-color: rgba(0, 0, 0, 0.25);
```

Comment:
@PhilippeBarre it doesn't look good. Can you mix up with the background?

Reply:
![image](https://user-images.githubusercontent.com/56122713/95024103-5619a080-0681-11eb-93eb-42da1132e2ad.png)

Hmm sorry, what doesn't look good ?

This background-color is the color of the toggle when it is unchecked. So it's black, transparent to see the image behind it.

Reply:
I will change the active color to accord with your theme
![image](https://user-images.githubusercontent.com/56122713/95024242-0ab3c200-0682-11eb-9d41-26e9f93fc19b.png)